tools/astarte_export/Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM elixir:1.11.4 as builder
+FROM hexpm/elixir:1.14.5-erlang-25.3.2-debian-bullseye-20230522-slim as builder
```

Comment:
Note that you are also updating Debian from stretch to bullseye

Reply:
I do not understand what you are trying to convey here. Would it be possible for you to clarify this for me?